single-inn_member.php

```diff
@@ -33,23 +33,23 @@
 			<ul class="social">
 				<?php
 					if ( ! empty( $meta['_facebook_url'][0] ) ) {
-						$url = "https://fb.com/" . $meta['_facebook_url'][0];
+						$url = $meta['_facebook_url'][0];
```

Comment:
Can we add `esc_url()` around the urls - so like
```$url = esc_url( $meta['_facebook_url'][0] );```
https://codex.wordpress.org/Function_Reference/esc_url